mlflow/store/rest_store.py

```diff
@@ -94,6 +96,7 @@ def get_experiment(self, experiment_id):
         Fetches the experiment from the backend store.
 
```

Comment:
Could you use the same verb tense throughout?  In models.py you use "Convert DB model".  Could this be "Fetch the experiment" and below "Fetch the run", etc?

Reply:
Done! And fixed the other ones in this file as well.